app/src/main/res/values/styles.xml

```diff
@@ -24,6 +24,10 @@
 		<item name="colorAccent">@color/accent</item>
 	</style>
 
+	<style name="AlertDialogTheme" parent="Theme.AppCompat.Dialog.Alert">
```

Comment:
Would it make sense to have AppTheme be the parent? Or do you need to inherit from the dialog theme?

Alternatively, would it make sense just to use R.style.AppTheme when you instantiate the AlertDialog.Builder objects?